contracts/contracts/NoLossDao.sol

```diff
@@ -50,13 +47,21 @@ contract NoLossDao is Initializable {
     _;
   }
 
-  modifier blankUser() {
-    require(depositedDai[msg.sender] == 0, 'Person is already a user');
+  modifier blankUser(uint256 amount) {
+    require(amount == 0, 'Person is already a user');
     _;
   }
 
   modifier userStaked(address givenAddress) {
-    require(depositedDai[givenAddress] > 0, 'User has no stake');
+    require(
+      depositContract.depositedDai(givenAddress) > 0,
+      'User has no stake'
+    );
+    _;
+  }
+
+  modifier userStakedSimple(uint256 amount) {
```

Comment:
The purpose of this, is when we call a function from poolDeposits, that needs this modifier, instead of calling itself back to get the value, we rather just pass it through and check the logic. The other modifier needs to remain cause there are some cases where the function calls aren't from the pool deposits contract